lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -121,8 +121,11 @@ function dosomething_metatag_get_permalink_vars() {
   $rbid = arg(1);
   $fid = ($_GET['fid']);
   $rb_entity = reportback_load($rbid);
-  if (!$fid) {
-    $fid = $rb_entity->fids[0];
+  if (!$fid && !empty($rb_entity->fids)) {
+    $fid = array_pop($rb_entity->fids);
+  }
+  else {
+    $fid = array_pop($rb_entity->getFids());
```

Comment:
two dpms, one from the end of `__construct` and the other inside `dosomething_metatag_get_permalink_vars`
![](http://cl.ly/image/1C072q3r0H3u/Screen%20Shot%202015-03-16%20at%205.06.34%20PM.png)